src/System.Net.Security/tests/FunctionalTests/ClientAsyncAuthenticateTest.cs

```diff
@@ -224,12 +215,12 @@ private void ClientAsyncSslHelper(SslProtocols clientSslProtocols, SslProtocols
             using (var client = new TcpClient(AddressFamily.InterNetworkV6))
             {
                 server.SslProtocols = serverSslProtocols;
-                client.Connect(server.RemoteEndPoint);
+                await client.ConnectAsync(server.RemoteEndPoint.Address, server.RemoteEndPoint.Port);
                 using (SslStream sslStream = new SslStream(client.GetStream(), false, AllowAnyServerCertificate, null))
                 {
-                    IAsyncResult async = sslStream.BeginAuthenticateAsClient("localhost", null, clientSslProtocols, false, null, null);
```

Comment:
The APM SslStream and Stream extensions are implemented within `SslStreamAPMExtensions.cs`.
If these changed, then we should also remove the extension classes.